ui/apps/platform/src/Containers/Dashboard/PatternFly/DashboardPage.tsx

```diff
@@ -41,10 +41,15 @@ function DashboardPage() {
             <Divider component="div" />
             <PageSection>
                 <Gallery
-                    style={{
-                        // Ensure the grid has never grows large enough to show 4 columns
-                        maxWidth: `calc(calc(${minWidgetWidth}px * 4) + calc(var(--pf-l-gallery--m-gutter--GridGap) * 3) - 1px)`,
-                    }}
+                    style={
```

Comment:
An observation about System Configuration which had inconsistencies in responsive layout:
* `Gallery` in non-editable view
* `Grid` in editable form

Because `Grid` props specify predictable number of columns for breakpoints, I replaced `Gallery` element.

I remember that dashboard layout changed from `Grid` to `Gallery` during development, but forget why.

Reply:
Ouch, that sounds like the other situation.